package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "autolabcling",
+  "version": "1.0.0",
+  "description": "Codebase for backward incompatible revision of autolabcli v0.1.1.",
+  "main": "index.js",
+  "scripts": {
+    "test": "nyc --reporter=lcov --reporter=text-lcov -s --all mocha 'test/**/*.js' > /dev/null",
+    "posttest": "nyc report --reporter=lcov > coverage.lcov && codecov"
+  },
+  "bin": {
+    "autolab": "./index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/1AmNegan/autolabcli.git"
```

Comment:
The URL needs to point to https://github.com/AutolabJS/autolabcli.git

Please correct at other places in the document too.